SYCL/Config/select_device.cpp

```diff
@@ -196,13 +192,28 @@ int main() {
                    << "}}" << std::endl;
                 passed = true;
                 break;
-              } else if ((plt.get_backend() ==
-                          backend::ext_oneapi_level_zero) &&
+              } else if (((plt.get_backend() ==
+                           backend::ext_oneapi_level_zero) ||
+                          (plt.get_backend() == backend::level_zero)) &&
```

Comment:
backend::level_zero, backend::hip and backend::cuda values are deprecated. According to SYCL spec we must use backend::ext_oneapi_level_zero, backend::ext_oneapi_hip and backend::ext_oneapi_cuda instead.

This commit did the renaming in the tests some time ago: intel@18c2f11

So I believe we shouldn't check for deprecated values in this test. These usages result in deprecated warnings as well.